inst/extdata/aglu/Muller_crops.csv

```diff
@@ -0,0 +1,21 @@
+# File: Muller_crops.csv,
+# Title: Mueller crop and GTAP crop mapping file,
```

Comment:
It would good to get a reference (`Source`) for "Mueller crop" in here if possible.